ext/File-Find/lib/File/Find.pm

```diff
@@ -954,12 +954,13 @@ C<$_> will be the same as C<$File::Find::name>.
 
 =item C<untaint>
 
-If find is used in L<taint-mode|perlsec/Taint mode> (-T command line switch or
-if EUID != UID or if EGID != GID), then internally directory names have to be
-untainted before they can be C<chdir>'d to. Therefore they are checked against
-a regular expression I<untaint_pattern>.  Note that all names passed to the
-user's C<wanted()> function are still tainted. If this option is used while not
-in taint-mode, C<untaint> is a no-op.
+If find is used in L<taint-mode|perlsec/Taint mode> (C<-T> command line
+switch or C<if EUID != UID> or C<if EGID != GID>), then internally
+directory names have to be untainted before they can be C<chdir>'d to.
+Therefore they are checked against a regular expression C<untaint_pattern>.
+Note that all names passed to the user's C<wanted()> function
+are still tainted.
+If this option is used while not in taint-mode, C<untaint> is a no-op.
```

Comment:
You might as well run line 963 into line 962, assuming they're both in the same paragraph.